coderd/database/migrations/000050_apikey_scope.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TYPE api_key_scope AS ENUM (
+    'all',
+    'application_connect'
+);
```

Comment:
If we add custom scopes later, this cannot be an enum. Roles are not enums.

Your call, but it isn't really necessary to enforce at the db level.

Reply:
I think we can resolve this problem with migrations when the time comes. Technically, the only correct and future proof option is to use UUIDs and foreign keys to a roles table but we're not at that point yet.